spec.html

```diff
@@ -35054,43 +35104,16 @@ <h1>Pattern Semantics</h1>
-        <p>The descriptions below use the following aliases:</p>
-        <ul>
-          <li>
-            _Input_ is a List whose elements are the characters of the String being matched by the regular expression pattern. Each character is either a code unit or a code point, depending upon the kind of pattern involved. The notation _Input_[_n_] means the _n_<sup>th</sup> character of _Input_, where _n_ can range between 0 (inclusive) and _InputLength_ (exclusive).
-          </li>
-          <li>
-            _InputLength_ is the number of characters in _Input_.
-          </li>
-          <li>
-            _NcapturingParens_ is the total number of left-capturing parentheses (i.e. the total number of <emu-grammar>Atom :: `(` GroupSpecifier Disjunction `)`</emu-grammar> Parse Nodes) in the pattern. A left-capturing parenthesis is any `(` pattern character that is matched by the `(` terminal of the <emu-grammar>Atom :: `(` GroupSpecifier Disjunction `)`</emu-grammar> production.
-          </li>
-          <li>
-            _DotAll_ is *true* if the RegExp object's [[OriginalFlags]] internal slot contains *"s"* and otherwise is *false*.
-          </li>
-          <li>
-            _IgnoreCase_ is *true* if the RegExp object's [[OriginalFlags]] internal slot contains *"i"* and otherwise is *false*.
-          </li>
-          <li>
-            _Multiline_ is *true* if the RegExp object's [[OriginalFlags]] internal slot contains *"m"* and otherwise is *false*.
-          </li>
-          <li>
-            _Unicode_ is *true* if the RegExp object's [[OriginalFlags]] internal slot contains *"u"* and otherwise is *false*.
-          </li>
-          <li oldids="sec-runtime-semantics-wordcharacters-abstract-operation">
-            _WordCharacters_ is the mathematical set that is the union of all sixty-three characters in *"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_"* (letters, numbers, and U+005F (LOW LINE) in the Unicode Basic Latin block) and all characters _c_ for which _c_ is not in that set but Canonicalize(_c_) is. _WordCharacters_ cannot contain more than sixty-three characters unless _Unicode_ and _IgnoreCase_ are both *true*.
-          </li>
-        </ul>
-        <p>Furthermore, the descriptions below use the following internal data structures:</p>
+        <p>The descriptions below use the following internal data structures:</p>
         <ul>
           <li>
-            A <em>CharSet</em> is a mathematical set of characters. When the _Unicode_ flag is *true*, &ldquo;all characters&rdquo; means the CharSet containing all code point values; otherwise &ldquo;all characters&rdquo; means the CharSet containing all code unit values.
+            A <em>CharSet</em> is a mathematical set of characters. When the _rer_.[[Unicode]] flag is *true*, &ldquo;all characters&rdquo; means the CharSet containing all code point values; otherwise &ldquo;all characters&rdquo; means the CharSet containing all code unit values.
```

Comment:
I don't think "_rer_" has a referent here.
```suggestion
            A <em>CharSet</em> is a mathematical set of characters. In the context of a Unicode pattern, &ldquo;all characters&rdquo; means the CharSet containing all code point values; otherwise &ldquo;all characters&rdquo; means the CharSet containing all code unit values.
```

Reply:
It seems like `_rer_` is behaving like one of these global aliases here. We could explicitly parameterize and say 

`&ldquo;All characters given _rer_&rdquo; where _rer_ is a RegExp Record means the CharSet containing all code point values when _rer_.[[Unicode]] is *true*; otherwise it means the CharSet containing all code unit values.`

Reply:
I'm going with @gibson042's suggestion; we don't need to bring up RegExp Records here.

Reply:
wfm